package.json

```diff
@@ -7,7 +7,8 @@
     "test": "jest",
     "testw": "jest --watch",
     "tsc": "tsc",
-    "start": "npx ts-node index.ts"
+    "start": "npx ts-node index.ts",
+    "scrap": "npx ts-node index.ts $HS_OPTIONS"
```

Comment:
Remove "start" script, and set "scrap" to "npx ts-node index.ts" since we're not using `HS_OPTIONS` anymore